serialization/serialization_config.go

```diff
@@ -26,6 +26,7 @@ import (
 type Config struct {
 	globalSerializer                    Serializer
 	customSerializers                   map[reflect.Type]Serializer
+	Compact                             CompactConfig
```

Comment:
why is this public, while the others are private?

Reply:
We've decided to have a separate `Compact` section to make it similar to all clients. `CompactConfig` has the private `serializers map[string]CompactSerializer` field and public `SetSerializers` method, similar to global/custom serializers.